test/mainnet/ynbtck.spec.sol

```diff
@@ -297,16 +299,50 @@ contract YnBTCkTest is Test, AssertUtils, MainnetKernelActors, EtchUtils, VaultU
     }
 
     function getBTCB(uint256 amount) internal {
-        vm.deal(bob, amount);
+        uint256 beforeBobBTCB = btcb.balanceOf(bob);
+        
+        vm.prank(0xF977814e90dA44bFA03b6295A0616a897441aceC);
```

Comment:
you can also use vm.deal(address(btcb), amount); in order to give an erc20 balance to bob.  one disadvantage of using the approach you use here is if the whale balance ever changes then it will break the tests.

Reply:
right, deal sometimes doesn't work let me try that